po/ko.po

```diff
@@ -13831,6 +13981,83 @@ msgid ""
 "```"
 msgstr ""
 
+#: src/basic-syntax/scalar-types.md:21
+msgid "There are a few syntaxes which are not shown above:"
+msgstr "위에 표시되지 않은 몇 가지 문법이 있습니다."
+
+#: src/basic-syntax/scalar-types.md:23
+msgid ""
+"- Raw strings allow you to create a `&str` value with escapes disabled: "
+"`r\"\\n\"\n"
+"  == \"\\\\\\\\n\"`. You can embed double-quotes by using an equal amount of "
+"`#` on\n"
+"  either side of the quotes:\n"
+"\n"
+"  ```rust,editable\n"
+"  fn main() {\n"
+"      println!(r#\"<a href=\"link.html\">link</a>\"#);\n"
+"      println!(\"<a href=\\\"link.html\\\">link</a>\");\n"
+"  }\n"
+"  ```\n"
+"\n"
+"- Byte strings allow you to create a `&[u8]` value directly:\n"
+"\n"
+"  ```rust,editable\n"
+"  fn main() {\n"
+"      println!(\"{:?}\", b\"abc\");\n"
+"      println!(\"{:?}\", &[97, 98, 99]);\n"
+"  }\n"
+"  ```"
+msgstr ""
+"- 원시 문자열을 사용하면 이스케이프되지 않는 `&str` 값을 만들 수 있습니"
+"다. `r\"\\n\"\n"
+"  == \"\\\\\\\\n\"` 따옴표의 양쪽에 같은 양의 `#`을\n"
+" 사용하여 큰따옴표를 삽입할 수 있습니다.\n"
+"\n"
+"  ```rust,editable\n"
+"  fn main() {\n"
+"      println!(r#\"<a href=\"link.html\">link</a>\"#);\n"
+"      println!(\"<a href=\\\"link.html\\\">link</a>\");\n"
+"  }\n"
+"  ```\n"
+"\n"
+"- Byte strings allow you to create a `&[u8]` value directly:\n"
+"\n"
+"  ```rust,editable\n"
+"  fn main() {\n"
+"      println!(\"{:?}\", b\"abc\");\n"
+"      println!(\"{:?}\", &[97, 98, 99]);\n"
+"  }\n"
+"  ```"
+
+#: src/basic-syntax/functions.md:5
+msgid ""
+"```rust,editable\n"
+"fn main() {\n"
+"    print_fizzbuzz_to(20);\n"
+"}\n"
+"\n"
+"fn is_divisible(n: u32, divisor: u32) -> bool {\n"
+"    if divisor == 0 {\n"
+"        return false;\n"
+"    }\n"
+"    n % divisor == 0\n"
+"}\n"
+"\n"
+"fn fizzbuzz(n: u32) -> String {\n"
+"    let fizz = if is_divisible(n, 3) { \"fizz\" } else { \"\" };\n"
+"    let buzz = if is_divisible(n, 5) { \"buzz\" } else { \"\" };\n"
+"    if fizz.is_empty() && buzz.is_empty() {\n"
+"        return format!(\"{n}\");\n"
+"    }\n"
+"    format!(\"{fizz}{buzz}\")\n"
+"}\n"
+"\n"
+"fn print_fizzbuzz_to(n: u32) {\n"
+"    for i in 1..=n {\n"
+"        println!(\"{}\", fizzbuzz(i));\n"
+"    }\n"
+
```

Comment:
I accidentally removed the no-op translation of this code block... let's remove the `msgid` field as well for now:

```suggestion

```